README.md

```diff
@@ -71,3 +74,15 @@ docker-compose up frontend
 ```
 
 See [frontend/README.md](frontend/README.md) for more documentation.
+
+### DB
+```bash
+docker-compose up db
+```
+
+This container will allow you to run a fully functioning local system with a `POSTGRES_URL` at `db.local`.
+
+The database will be initialized but won't contain any data. There is some sample data included in this repository in the `data` directory as well as a script to load the data into the database. With the DB container running, on the `backend` container, run:
+```bash
```

Comment:
can this be a make script? does it have to be run every time you spin up the db? can you also clarify in the text above that it's for rhode island (even though it's in the filename, good to be clear)